tools/manifestfile.py

```diff
@@ -269,9 +348,12 @@ def include(self, manifest_path, top_level=False, **kwargs):
             if options.extra_features:
                 # freeze extra modules.
         """
+        if is_require:
+            self._metadata[-1].check_initialised(self._mode)
+
         if not isinstance(manifest_path, str):
             for m in manifest_path:
-                self.include(m)
+                self.include(m, **kwargs)
```

Comment:
should it pass through `is_require=is_require`?  previously this would have been `top_level=False` (the default), so maybe it should be `is_require=True`?

Reply:
I also fixed the `pypi` kwarg here. Had it defaulting to `""` but it should be `None`.

Reply:
OK